### ISSUE TITLE
Source Code view - Feature Request

### ISSUE THREAD

**Issue by tariqrahiman** [mljar/studio#52]
Is it possible to add a new feature under View which will display the Source Code only for the Steps we added ?
This way if Users have existing code, it can forward engineer into Steps View - that would be a good feature.

However, as a start, a source code only view would help :-)
~ Tariq Rahiman

**Comment by pplonski** [comment]
What do you mean by Source Code for Steps? 
- is it Python source code for notebook
- or is it source code for Steps available in GUI? Would you like to dynamically define new steps?

**Comment by tariqrahiman** [comment]
> is it Python source code for notebook
Yes, this way, users can run any python notebook using the MLJar Studio. This helps in first time users able to run programs without going into adding steps visually. 

> or is it source code for Steps available in GUI? Would you like to dynamically define new steps?
This is another feature. However this is low priority, since your idea is to get users to use the GUI.
Hope this makes sense.

**Comment by pplonski** [comment]
You can just copy and paste the code into the cell. Or is it something different?

Anyway, @tariqrahiman what do you think about GUI for code generation after playing with it for a while?

**Comment by tariqrahiman** [comment]
@pplonski It is really awesome to work with this. So easy to create steps and move forward. I did the full program in a few minutes and it is time saving. I am loving it.
